src/operations/finances.md

```diff
@@ -0,0 +1,89 @@
+Aside from out-of-pocket spending from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).
+
+## Internet Society Grant
+Internet Society was involved in discussions since ideation phases of the Toronto Community Network, and that is reflected in our [Proposal to Establish a Toronto Community Network](https://github.com/tomeshnet/toronto-community-network/blob/master/operations/published-documents/proposal-draft-v003.pdf).
+This eventually led to a grant focused on deployment of high-capacity supernodes, which will establish an initial backbone from which to grow and sustain the Toronto Community Network.
+The initial supernodes span several major neighborhoods across the City of Toronto, selected and scheduled according to opportunities presented by the City of Toronto and our other collaborators through the [Digital Canopy project](https://newsroom.cisco.com/press-release-content?type=webcontent&articleId=2095927).
+
+The grant totals CAD 13,525 with the following budget:
+- $10,428.00 for hardware, covering 4 supernode deployments
+- $1,744.50 for operational expenses, including data centre colocation and internet transit
+- $1,352.00 for grant administration, a 10% fee retained by our fiscal host [Free Geek Toronto](https://freegeektoronto.org)
+
+The funds are payable to Free Geek Toronto in two installments, CAD 7,000 followed by CAD 6,525 upon completion of the milestone where 2 supernodes have been deployed.
+Toronto Mesh estimated a contribution of 1630 member hours of volunteer labour to the project.
+
+Toronto Mesh and Free Geek Toronto are expected to deliver 3 reports by the following dates:
+- 1st report: September 30, 2020
+- 2nd report: November 15, 2020
+- Final report: January 31, 2020 (summarizing all the work done until December 31, 2020)
+
+Four core members of Toronto Mesh, who will serve as Administrators for this grant (explained below), have signed a Memorandum of Understanding (MoU) with Free Geek Toronto to formalize the fiscal hosting relationship.
+The full Internet Society Partnership Agreement and MoU are [archived on our trusted drive](https://cloud.tomesh.net/apps/files/?dir=/tomeshnet-trusted/ISOC), accessible only to core members due to confidentiality requirements from our partners.
+
+### Administration
+The first of two installments in the amount of CAD 7,000 is paid to Free Geek Toronto in September of 2020, and the balance of CAD 6,300 (after 10% of administrative fees) is to be distributed for the project according to the budget set out in the grant proposal.
+We have two-levels of approvals for fund allocations:
+
+1. Four members of Toronto Mesh involved in the grant application are selected as Administrators, and will approve allocations according to whether an expense fits with the spirit of the grant objectives and its proposed budget.
```

Comment:
I would like to propose for @darkdrgn2k @TimTor @Shrinks99 and myself as the Administrators of this grant from Toronto Mesh. Is this okay with you guys?

@ASoTNetworks @makeworld-the-better-one you two are the people who were involved in the original TCN proposal, I want to know if this arrangement is okay with you guys too.

Reply:
This makes sense to me, I agree with the points about Hypha and I'm happy to sign it!

Reply:
This arrangement works for me, thanks!

Reply:
Sounds good to me